bot/cogs/moderation/infractions.py

```diff
@@ -230,7 +230,7 @@ async def action() -> None:
 
         await self.apply_infraction(ctx, infraction, user, action())
 
-    @respect_role_hierarchy()
+    @respect_role_hierarchy(2)
```

Comment:
This call makes sense to me as I've just reviewed the previous commit. However, I'm sure that if I come back to this in a few weeks, I'm going to be puzzled by what the `2` means in the decorator factory call. Is there a way we could use a nice keyword argument here? I know you don't like long names, so something shorter than `keyword_or_positional_argument`, and maybe even the current `name_or_pos` would do.

Reply:
agreed. this is very puzzling to me.